packages/react/src/components/tabs/Tabs.tsx

```diff
@@ -6,18 +6,39 @@ import styles from './Tabs.module.scss';
 import classNames from '../../utils/classNames';
 import { TabsContext } from './TabsContext';
 import { FCWithName } from '../../common/types';
+import { useTheme } from '../../hooks/useTheme';
+
+export interface TabsCustomTheme {
+  '--tablist-border-color'?: string;
+  '--tablist-border-size'?: string;
+  '--tab-color'?: string;
+  '--tab-font-size'?: string;
+  '--tab-height'?: string;
+  '--tab-min-width'?: string;
+  '--tab-spacing'?: string;
```

Comment:
Do we need to open these properties for customization? As they might affect accessibility somehow for example.

Reply:
One use case I can think of is that if someone uses tabs in some custom component that requires more changes to tab visuals. In this case, I would prefer that they use the HDS component even though visually modified one. These properties would allow the customisation if needed.

Reply:
Yes, I think you can create some unaccessible modifications with these properties. But then again, you can do that with the colour properties alone so I don't think that is a big issue.

Reply:
We decided to leave them as they are now